packages.json

```diff
@@ -18504,5 +18504,17 @@
     "description": "Nim CMAES library",
     "license": "Apache-2.0",
     "web": "https://github.com/zevv/nimcmaes"
+  },
+  {
+    "name": "soundex",
+    "url": "https:///",
```

Comment:
That doesn't look correct for me :P

Reply:
Thank you for the review.
I fix it and submit again.

Reply:
I checked about URL setting but I don't know how to set the URL.
How should I set the URL.

If it's not too much trouble, can you tell me?